db/re/item_db.conf

```diff
@@ -61939,6 +61940,11 @@ item_db: (
 	Slots: 1
 	Loc: 256
 	View: 525
+	Script: <"
+		bonus bDex, 2;
+		skill TF_DOUBLE, 5;
```

Comment:
@iRagno you are wrong here. The Hat is supposed to give you Level 5 Double Attack (+ your own skill level in it). Sidewinder Card just gives a chance to cast it, while the hat actually gives you the skill. This is something entirely different.
Asheraf is right here.